docs/docs/dev_docs/getting_started/sandbox.md

```diff
@@ -350,6 +350,47 @@ Our complete output should now be:
 
 That's it! We have successfully deployed a private token contract to an instance of the Aztec network and mined private state-transitioning transactions. We have also queried the resulting state all via the interfaces provided by the contract.
 
+## Updating
+To update the sandbox, simply run the curl command again:
+
+```sh
+/bin/bash -c "$(curl -fsSL 'https://sandbox.aztec.network')"
+```
+
+It will download and start the latest version of sandbox.
+
+If you would like to use a fixed version of sandbox you can export the `SANDBOX_VERSION` environmental variable.
+The version in our case is a tag of the `aztec-sandbox` image on [dockerhub](https://hub.docker.com/r/aztecprotocol/aztec-sandbox/tags).
+This is the full command:
+
+```sh
+export SANDBOX_VERSION=v0.7.0
+```
+
+If this variable is not set the latest version will be used.
+
+To manually specify that you want to use the latest version use the `latest` tag:
+
+```sh
+export SANDBOX_VERSION=latest
+```
+
+Now when you run the curl command it will use the version you specified.
+To verify that it's the case check the console output of the curl command.
+Alternatively you can open a new terminal and use aztec-cli to get the version.
+  
+```sh
+aztec-cli get-node-info
+```
+
+:::note
+
+Versions of `sandbox` and `aztec-cli` must match.
```

Comment:
Please could you include an instruction for how a user can query which version of the sandbox they're currently running? `echo $SANDBOX_VERSION` might not work in a new shell, so is there a query that can be sent to the sandbox, via the aztec-cli (for example)?